integration-tests/narayana-jta/src/main/resources/application.properties

```diff
@@ -8,4 +8,7 @@ quarkus.console.color=false
 quarkus.log.category."com.arjuna".level=INFO
 quarkus.log.category."io.quarkus".level=INFO
 
-quarkus.transaction-manager.object-store-directory=target/tx-object-store
\ No newline at end of file
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.transactions=disabled
```

Comment:
I notice below that you have created a class called `JdbcObjectStoreTestProfile`.
I am not familiar with `QuarkusTestProfile` so is this integration test activated by `JdbcObjectStoreTestProfile`? If not how is the integration test enabled and how does it relate to TransactionJdbcObjectStoreTest``

Reply:
Yeah, `QuarkusTestProfile` will force to start a new instance application for testing. We override some properties to enable the jdbc object-store. So now there are two integration tests:

- `TransactionalTestCase` which is testing for normal file system object store.
- `TransactionalJdbcObjectStoreTestCase` which is testing for jdbc object store.

Anyway current test is very simple to just verify the transaction status. And I think more funtional tests could be in `quarkus-qe`.

Reply:
Yeah, `QuarkusTestProfile` will force to start a new application for testing. We just override some properties to enable jdbc object store. So there are two integration tests:
- `TransactionalTestCase` which is testing for normal file system object store.
- `TransactionalJdbcObjectStoreTest` which is testing for jdbc object store.

Anyway, the current test is very simple to just verify the transaction status. I think the more funtinal tests could be in `quarkus-qe`.

Reply:
The default jdbc object store uses the `h2` database in this test. It sets `quarkus.datasource.jdbc.transactions=disabled` and I belive that the datasource which jdbc object store is using should not invovle in any transaction.

Reply:
What does the property `quarkus.datasource.jdbc.transactions` do and when would it be enabled?

Reply:
This propery is used by `quarkus-agroal` and there are three values:
- ENABLED
- XA
- DISABLED

see https://quarkus.io/guides/datasource#quarkus-agroal_quarkus.datasource.jdbc.transactions

NOTE, this is a default datasource configuration for using in `quarkus.transaction-manager.object-store.datasource`